docs/relying-parties/how-twos/google-iap.md

```diff
@@ -0,0 +1,109 @@
+
+A SubPlat engineer with developer access to Google Play Console (see [Prerequisites](#prerequisites)) can configure this for you when ready.
+
+Google Play Developer API keys are needed for SubPlat to make calls to Google API. See [Google IAP Integration](/ecosystem-platform/tutorials/subscription-platform#google-iap-integration) for more information on how to obtain these keys.
+
+### Send Real-time developer notifications to SubPlat
+
+A SubPlat engineer with developer access to Google Play Console (see [Prerequisites](#prerequisites)) can configure this for you when ready.
+
+The Sandbox and, when ready, Production endpoints in Real-time developer notifications must be updated with SubPlat's notification endpoint in order for our system to receive Real-time developer notifications.
+
+### Map Google IAP to Stripe plans
+
+In order to know what capabilities to grant a given Google IAP user, we map Play Store `productId`s to [Stripe `price` or `plan`](https://support.stripe.com/questions/how-to-create-products-and-prices) IDs (`price`s supersede `plan`s).
+
+1. Go into the Google Play Console and find your android app’s human-readable `productId`s.
+2. In the Stripe dashboard for the desired environment (stage or production), create a new, valid product in Stripe and a new, valid plan underneath that product.
+   - We hope to simplify this process in SubPlat 3.0, expected in 2023; see this [Jira comment](https://mozilla-hub.atlassian.net/browse/PSP-64?focusedCommentId=627245) for more details.
+3. Add each Play Store `productId` as a comma-separated value to a new `playSkuIds` metadata property on the newly created plan.
+
+### Prevent Google IAP subscribers from double subscribing
+
+As noted previously, we don’t support upgrades or any plan changes for Google IAP subscribers. However, it is possible for a Google IAP subscriber to try to sign up again for your product on the SubPlat website. Follow these steps to prevent them from double subscribing.
+
+1. In the Stripe dashboard for the given environment (stage or production), remove any `productSet` properties that may have been previously set on the newly created product/plan from [Map Google IAP to Stripe plans](#map-google-iap-to-stripe-plans).
+2. Add a `productSet` metadata key with a value equal to the comma-separated list of all the unique `productSet`s for all your product’s plans in the environment to the new product referenced in Step 1. Without their own `productSet` specified, the new plan will inherit its product’s `productSet`.
+
+## SubPlat API Calls
+
+These steps can be done by a developer and in parallel with the steps in [Configuration](#configuration).
+
+As mentioned in the [Introduction](#introduction), SubPlat has read-only access to Play Store subscription information. We will store Google IAP subscribers' data in our system and update the information when we receive notifications from the Play Store server, broadcasting updates in real time to the relevant RPs.
```

Comment:
I don't know if Google calls their notification server the "Play Store Server" FWIW like Apple calls their server the "App Store Server". Google calls these notifications RTDNs, so looking up those docs should clarify that.

Reply:
Good catch! I missed this during a find and replace for _App Store_. Fixed.